shrine/README.md

```diff
@@ -0,0 +1,45 @@
+# Shrine
+
+A sample shopping app that uses Material widgets in its UI and
+[`scoped_model`](https://pub.dartlang.org/packages/scoped_model) to
+manage the state of its shopping cart.
+
+## Goals
+
+* Show how to customize Flutter's Material Compoenent widgets to produce
+  a unique design for an app.
+* Show how to use `scoped_model` to manage an app's state and access it
```

Comment:
There was a cool video on scoped model architecture for Flutter right? Could you include a link?

Reply:
I don't think we have anything specifically for scoped_model. There was an I/O talk that showed it as one of a few options, but I think that's about it.